content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing parts of code to verify its behavior.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
```

Comment:
```suggestion
  - 'learn-javascript-unit-testing'
  - 'paths/front-end-engineer-career-path`
```


Ideally, we want each entry to have two `CatalogContent` slugs: one `learn-` and one `path-`. General entries can be tricky to nail down in this regard but I found a slug for the Front-end Career Path on Codecademy that might be relevant since there are some lessons related to testing in there.